libraries/AP_Motors/AP_Motors6DOF.cpp

```diff
@@ -338,6 +343,42 @@ void AP_Motors6DOF::output_armed_stabilizing()
             }
         }
     }
+
+	// Current limiting
+    if (_batt_current_max <= 0.0f) {
+        return;
+    }
+
+    static float _output_limited = 1.0f;
+    static float _batt_current_last = 0.0f;
```

Comment:
Can you change these to be in the class instead? It's the preferred way now.

Reply:
done